### DISCUSSION TITLE
Reduce power

### DISCUSSION THREAD

**Discussion by Medvidas** [egzumer/uv-k5-firmware-custom#52] [General]
Hi,
most of the radios like UV-K5 are used on the free PMR bands. 
But according to the radio specifications, even the LOW power
still emitts 1,5 W, while PMR permited power should be less than 0,5 W. 

Is it possible to anyhow lower the LOW power for usage within PMR band?

Thank you, Ales

**Comment by egzumer** [comment]
It is even worse than that. As far as I know there is very little difference between power levels on this radio. It is possible do decrease the power. I believe 1o11 already implemented that, I don't know what the results are. However even if you decrease the power output it is technically still illegal to transmit on PMR with this radio as it has detachable antenna.

**Comment by wmarkow** [comment]
According to [amnemonic/Quansheng_UV-K5_Firmware](https://github.com/amnemonic/Quansheng_UV-K5_Firmware/tree/main/hardware) repository, the device has BK4819 chip on board. Then taking a look at the [BK4819(V3) Application Note 20210428.pdf](https://github.com/amnemonic/Quansheng_UV-K5_Firmware/blob/main/hardware/BK4819(V3)%20Application%20Note%2020210428.pdf) page 14, the internal power amplifier of BK4819 can be set from -23.7dBm to 7.26dBm only. On the other side the device can transmit with max 5W, which is around 37dBm, so I guess that the device has an external power amplifier around 30dBm (together with 7.26dBm of BK4819 gives 37dBm of max output power). The minimal output power should be then -23.7dBm + 30dBm which is 6.3dBm, which is around 4mW. The original software has only three power level (low, medium and high) however it should be possible (with custom software) to make more options here. 
Please correct me.

**Reply by egzumer** [reply]
Thanks for pointing out those informations. More power levels would be nice. The problem is that both radio and configuration softwares are designed for 3 power levels only. It would make it incompatible. It  can still be implemented but that has to be taken into account.
Second, each radio has it's own factory calibration. The calibration values are different on each radio. I don't know how accurate the calibration is, doesn't seem to be very accurate, as differences between power levels can be minimal as I saw on some tests on youtube. I don't know what is the variation between different units, implementing one solution for all radios is problematic because of that.
Third problem is that I don't have power meter.

**Comment by ErikS-web** [comment]
The indicated values are maximum values for the frequency for which the radio is intended (2m and 70 cm).

Have you already tried to measure the transmitted power on a PMR frequency? Perhaps it is already lower there.

**Comment by Medvidas** [comment]
Thank you for your reactions guys.

I thing that this radio is very popular across non-ham users and at the end, they will use it mainly in PMR bands I guess.
So from this perspective better PMR compliance will be beneficial. 
I agree that still not legal PMR station because of removable antena, 
but I think that anybody understand that limited transmit power is the key factor here.

As Wmarkow pointed out, we may play with the power levels of BK4819 in wide scale.
I think that original 5W/3W/1.5W (from K5/6 specs.) power levels are not the best option. 
My primary Yaesu FT-60 has 5W/2W/0.5W which is more flexible. 
As you know, it is always better to use the lowest workable transmit power from various reasons.

Lower LOW power settings can not only help PMR operations, but also substantialy reduce power consumption of the radio, 
and that can be great.

**Comment by ErikS-web** [comment]
The VSWR of the antenna will also not be good. Unfortunately I can only measure the Power and VSWR up to 150 MHz.

**Comment by Medvidas** [comment]
This is how antenna is looking on my NanoVNA:
![20231111_212844_cr](https://github.com/egzumer/uv-k5-firmware-custom/assets/57909431/21e900b6-2fff-4a88-ac64-2d4079a464bf)
![20231111_212913_cr](https://github.com/egzumer/uv-k5-firmware-custom/assets/57909431/28ee6e86-ffec-48eb-bc00-5b3a22ba3f0b)

I would say, not as bad as many think, esp. on VHF.

**Reply by ErikS-web** [reply]
Nice.

And for PMR 1 on 446.00625

**Comment by Medvidas** [comment]
Right, it is 2.4 on PMR 1 freq. That means only 83 % of power is radiated - 4150 mW from 5000 mW max.

**Comment by ErikS-web** [comment]
Use this Build option to make _your own dedicated Lower Power firmware_

See the options in the `README.md` under [user-customization.](https://github.com/egzumer/uv-k5-firmware-custom/tree/a43a3f949f32ed8ffbf0438face0edb7d093811e#user-customization)

**ENABLE_REDUCE_LOW_MID_TX_POWER:= 0**       _makes medium and low power settings even lower_
It is already designed, but not active (set 0) in the Release(s).

When set 1 it reduces the low and mid TX power levels (high remains unchanged). Not specefic PMR-frequencies !

So if you want to use it. Customize and Build your firmware with the instructions

**Comment by ErikS-web** [comment]
TIP. If you want to limit the (TX-)range. Its is much easier to use a 'bad' antenna ; usually this is a small antenna ;-)

**Reply by egzumer** [reply]
Yeh, the best "solution" is to unscrew it, or even better short the terminals.

**Comment by ErikS-web** [comment]
Have a look at this video... see my reaction in #168

The prove what the radio is designed for.

**Comment by AsciiWolf** [comment]
Just a note: Everyone seems to discuss PMR446 here, but the reduced output is not only for PMR. For example, in our country, we have license-free channels (on both 2m and 70cm) that allow use of custom handhelds with detachable antennas, but some of these channels allow only 1W of power. Reduced low TX power level on Quansheng would be a significant help here since it would allow to legally use a Quansheng on these freqs. Also, it would help with battery usage as well. 1W is perfectly sufficient for many local direct contacts and even for some ham repeater ones.

**Comment by ErikS-web** [comment]
What country ?

**Reply by AsciiWolf** [reply]
Czech Republic. It's called "shared frequencies" here.

**Reply by Medvidas** [reply]
There are more unlicenced or combined bands like that:
It is U.S. FRS and GMRS band (462/467 MHz) using some 0.5 and 2W channels. Similar in Canada and Mexico.
Then MURS bands in another regions, limited to 2 W.
German Freenet in 149 MHz band limited to 1 W.
South Korea UHF-FM 448.750–449.2625 MHz band limited to 0.5 W.
In Norway KDR 444 with 2 W limit.
China 409 MHz general public band with 0.5 W limit.
...
To me 0.5 W, 2 W  and 5 (or max. W) seems like the sweet spot for most unlicenced and HAM radio band usage from legislative and also battery endurance perspective.
